doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -297,6 +297,8 @@ Cellular samples
 * :ref:`nrf_cloud_multi_service` sample:
 
   * Fixed an issue with an uninitialized variable in the :c:func:`handle_at_cmd_requests` function.
+  * Fixed an issue with the :kconfig:option:`CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE` Kconfig value
+    in the :file:`overlay-coap_nrf_provisioning.conf` file: the value was too small.
```

Comment:
Please formulate as follows:

* Fixed:

  * An issue with an uninitialized variable in the :c:func:`handle_at_cmd_requests` function.
  * An issue with the too small :kconfig:option:`CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE` Kconfig value
    in the :file:`overlay-coap_nrf_provisioning.conf` file.